### PR TITLE
[python3/en] Some improvements to Python 3

### PR DESCRIPTION
**Pull request by nedbat**
- [ ] I solemnly swear that this is all original content of which I am the original author
- [ ] Pull request title is prepended with `[language/lang-code]`
- [ ] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: python3.html.markdown [hunk 1]
```diff
@@ -381,7 +379,7 @@ filled_set | other_set  # => {1, 2, 3, 4, 5, 6}
 # Let's just make a variable
 some_var = 5
 
-# Here is an if statement. Indentation is significant in python!
+# Here is an if statement. Indentation is significant in Python! Use four spaces.
```
Comment (vendethiel) [inline python3.html.markdown]: "PEP8 recommends four spaces" would be better

Comment (nedbat) [inline python3.html.markdown]: I'm not sure the audience understands what PEP8 is.  Elsewhere, we say, "Convention is to use lower_case_with_underscores", so how about "Convention is..."

Comment (vendethiel) [inline python3.html.markdown]: Sounds good, thanks.

**Comment by vendethiel** [comment]
Thanks

**Pull request merged**